pkg/client/imagestreams.go

```diff
@@ -139,7 +140,7 @@ func transformUnsupported(err error) error {
 	// enabled by policy. A create request will return a Forbidden(403) error.
 	// We want to return ErrImageStreamImportUnsupported to allow fallback behavior
 	// in clients.
-	if apierrs.IsForbidden(err) {
```

Comment:
Yes, quota denied is 403 Forbidden. Without it, the `oc import-image` will ignore denied error from `is.Import()` and will continue with image stream update/create and will hang on `waitForImport()`. With this change, client gives up and terminates.

Making an unsupported error out of quota denied seems wrong to me. What do you suggest instead? Make current client more tolerant without knowing, what really happened and let him report some generic failure?